features/flows/classification/engine/impl/src/main/java/org/opennms/netmgt/flows/classification/internal/DefaultClassificationEngine.java

```diff
@@ -110,9 +110,9 @@ public void reload() {
                     .append("minComp  : " + tree.info.minComp).append('\n')
                     .append("maxComp  : " + tree.info.maxComp).append('\n')
                     .append("avgComp  : " + (double) tree.info.sumComp / tree.info.leaves).append('\n')
-                    .append("minRuleSetSize :" + tree.info.minRuleSetSize).append('\n')
-                    .append("maxRuleSetSize :" + tree.info.maxRuleSetSize).append('\n')
-                    .append("avgRuleSetSize :" + (double) tree.info.sumRuleSetSize / tree.info.leaves).append('\n');
+                    .append("minLeafSize : " + tree.info.minLeafSize).append('\n')
```

Comment:
Unnecessary refactoring. Please extract in single commit next time (read about `git add -p -i` to split these changes in multiple commits).